static/js/compose.js

```diff
@@ -451,23 +451,33 @@ exports.validate_stream_message_address_info = function (stream_name) {
         return true;
     }
 
+    var context = {
+       link: "<a href='#streams/all'>",
+       para: '<p>',
+       bold: '<b>',
+       closepara: '</p>',
+       closelink: '</a>',
+       closebold: '</b>',
+       interpolation: {prefix: "{{", suffix: "}}", escapeValue: false},  // We override the interpolation prefix and suffix here.
+    };
```

Comment:
This feels very hacky; is there a better way to do this with the i18next API?

Reply:
Yeah, I think that approach makes more sense.  I'm closing this; @umairwaheed can you submit a PR doing this correct?

Reply:
Yeah.